packages/experimental/Dropdown/src/Dropdown/Dropdown.styling.ts

```diff
@@ -0,0 +1,6 @@
+import { UseStylingOptions } from '@fluentui-react-native/framework';
+import { dropdownName, DropdownProps, DropdownSlotProps, DropdownTokens } from './Dropdown.types';
+
+export const stylingSettings: UseStylingOptions<DropdownProps, DropdownSlotProps, DropdownTokens> = {
+  tokens: [dropdownName],
```

Comment:
The name is a token?

Reply:
Yeah, it generally has been if you look at other components. Why? not sure, maybe we shouldn't be doing it? or maybe it helps with some sort of framework lookup stuff.